### PR TITLE
The first fix for #500

### PR DESCRIPTION
**Pull request by ravil-mobile**

### CODE REVIEW

File: src/DynamicRupture/Output/Builders/PickPointBuilder.hpp [hunk 1]
```diff
@@ -107,8 +107,8 @@ class PickPointBuilder : public OutputBuilder {
                                                           meshVertices[element.vertices[3]].coords,
                                                           receiver.global.getAsEigenVector());
         receiver.isInside = true;
-
       } else {
+        logInfo() << "pickpoint fault output: receiver (" << receiverId << ") is not inside";
```
Comment (Thomas-Ulrich) [inline src/DynamicRupture/Output/Builders/PickPointBuilder.hpp:111]: Can you clarifiy:
is not inside what? The mesh? volume cells adjacent to the faults?

**Review by Thomas-Ulrich — APPROVED** [review]
looks good

**Comment by sebwolf-de** [comment]
What do you mean with first?

**Comment by sebwolf-de** [comment]
Also, did you have a look at: [37ec934](https://github.com/SeisSol/SeisSol/commit/37ec934692734f99d33e9c69e511f7252d253f89)?

File: src/DynamicRupture/Output/Builders/PickPointBuilder.hpp (hunk 1 above)
Comment (ravil-mobile) [reply src/DynamicRupture/Output/Builders/PickPointBuilder.hpp]: Added the following:
```
        logInfo() << "pickpoint fault output: receiver (" << receiverId << ") is not inside"
                  << "of any element along the rupture surface";
````
Does it sound better?

Comment (Thomas-Ulrich) [reply src/DynamicRupture/Output/Builders/PickPointBuilder.hpp]: ok.

**Comment by ravil-mobile** [comment]
> Also, did you have a look at: [37ec934](https://github.com/SeisSol/SeisSol/commit/37ec934692734f99d33e9c69e511f7252d253f89)?

I think these line are not correct:
```
          const size_t faultVertexIdx = vertexIdx + 4 * elementIdx;
          faultVertices[faultVertexIdx] = meshVertices[element->vertices[vertexIdx]];
```

I worte:
```
          const size_t faultVertexIdx = vertexIdx + 4 * (sideId + numSides * faultIdx);
          faultVertices[faultVertexIdx] = meshVertices[element->vertices[vertexIdx]];
```

where `constexpr int numSides{2};`

what do you think?

**Pull request merged**